demisto_sdk/commands/find_dependencies/find_dependencies.py

```diff
@@ -236,11 +240,18 @@ def _search_packs_by_items_names_or_ids(items_names: Union[str, list],
             items_names = [items_names]
 
         for item_name in items_names:
-            item_possible_names = [item_name, f'incident_{item_name}', f'indicator_{item_name}', f'{item_name}-mapper']
+            if incident_or_indicator == 'Incident':
+                item_possible_ids = [item_name, f'incident_{item_name}', f'{item_name}-mapper']
+            elif incident_or_indicator == 'Indicator':
+                item_possible_ids = [item_name, f'indicator_{item_name}', f'{item_name}-mapper']
+            elif incident_or_indicator == 'both':
+                item_possible_ids = [item_name, f'incident_{item_name}', f'indicator_{item_name}',
+                                     f'{item_name}-mapper']
+
             for item_from_id_set in items_list:
                 machine_name = list(item_from_id_set.keys())[0]
                 item_details = list(item_from_id_set.values())[0]
-                if (machine_name in item_possible_names or item_name == item_details.get('name')) \
+                if (machine_name in item_possible_ids or item_name == item_details.get('name')) \
```

Comment:
Why did we change this?

Reply:
https://github.com/demisto/demisto-sdk/pull/1290#discussion_r650486686